CONTRIBUTING.md

```diff
@@ -16,46 +16,85 @@ In addition to Github issues, we also utilize a feedback site that helps our pro
 
 We have a 3 step process for contributions:
 
-1. Sign or be added to an existing [Contributor License Agreement (CLA)](https://supermarket.chef.io/become-a-contributor).
-2. Create a Github Pull Request for your change.
-3. Perform a [Code Review](#cr) with the project maintainers on the pull request.
+1. Commit changes to a git branch, making sure to sign those changes for the [Developer Certificate of Origin](#developer-certification-of-origin-dco).
+2. Create a Github Pull Request for your change, following the instructions in the pull request template.
+3. Perform a [Code Review](#code-review-process) with the project maintainers on the pull request.
 
 ### Pull Request Requirements
 
 Chef Projects are built to last. We strive to ensure high quality throughout the experience. In order to ensure this, we require that all pull requests to Chef projects meet these specifications:
 
 1. **Tests:** To ensure high quality code and protect against future regressions, we require all the code in Chef Projects to have at least unit test coverage. We use [RSpec](http://rspec.info/) for unit testing.
-2. **Green CI Tests:** We use [Travis CI](https://travis-ci.org/) and [AppVeyor](https://www.appveyor.com/) CI systems to test all pull requests. We require these test runs to succeed on every pull request before being merged.
+2. **Green CI Tests:** We use [Travis CI](https://travis-ci.org/) and/or [AppVeyor](https://www.appveyor.com/) CI systems to test all pull requests. We require these test runs to succeed on every pull request before being merged.
 
 ### Code Review Process
 
-Code review takes place in Github pull requests. See [this article](https://help.github.com/articles/using-pull-requests) if you're not familiar with Github Pull Requests.
+Code review takes place in Github pull requests. See [this article](https://help.github.com/articles/about-pull-requests/) if you're not familiar with Github Pull Requests.
 
-Once you open a pull request, Chef engineers will review your code and respond to your pull request with any feedback they might have. The process at this point is as follows:
+Once you open a pull request, project maintainers will review your code and respond to your pull request with any feedback they might have. The process at this point is as follows:
 
-1. 2 thumbs-ups are required from project maintainers or core contributors. See the master maintainers document for Chef projects at <https://github.com/chef/chef/blob/master/MAINTAINERS.md>.
+1. Two thumbs-up (:+1:) are required from project maintainers. See the master maintainers document for Chef projects at <https://github.com/chef/chef/blob/master/MAINTAINERS.md>.
 2. When ready, your pull request will be tagged with label `Ready For Merge`.
```

Comment:
At this point the second approving maintainer probably just clicks the merge button. But I'm not sure we need the nuts and bolts of that here. Eh, anyway, not sure how we're using this label anymore.